<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerClass</class>
 <widget class="QMainWindow" name="ViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QAxWidget" name="axRDPClientWidget">
    <property name="geometry" stdset="0">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>581</height>
     </rect>
    </property>
    <property name="sizePolicy" stdset="0">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>70</y>
      <width>135</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnControl">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>130</y>
      <width>135</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGetConnectionString">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>135</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Get connection</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="viewer.qrc"/>
 </resources>
 <connections/>
</ui>
